src/config.rs

```diff
@@ -78,6 +78,10 @@ pub(crate) struct RelabelConfig {
 #[derive(PartialEq, Eq, Debug, serde::Deserialize)]
 pub(crate) struct PrioritizeConfig {
     pub(crate) label: String,
+    #[serde(default)]
+    pub(crate) prioritize_on: Vec<String>,
+    #[serde(default)]
+    pub(crate) priorities_pattern: String,
```

Comment:
Maybe call that `priority_labels` and make it a `Vec<String>`, just in case other repos want to use this and don't have a pattern?

Reply:
Renamed it but left it as `String` because I don't think we are going to have complex patterns to identify labels really.